internal/common/gamestate/gamestate.go

```diff
@@ -15,7 +17,9 @@ type GameState struct {
 
 	// ClientInfos map from the shared.ClientID to ClientInfo.
 	// EXTRA note: Golang maps are made to be random!
-	ClientInfos map[shared.ClientID]ClientInfo
+	ClientInfos    map[shared.ClientID]ClientInfo
+	Environment    *disasters.Environment
+	DeerPopulation foraging.DeerPopulationModel
```

Comment:
You'll need to edit `Copy` for gameState. Unfortunately unlike python's `deepcopy` we have to do it manually for golang. structs are passed by reference in golang (same as objects in python) and hence the reference will be copied by default.

Reply:
Ah okay - thanks for the heads up!